src/Markdig.Tests/Specs/GridTableSpecs.md

```diff
@@ -4,9 +4,10 @@ This section describes the different extensions supported:
 
 ## Grid Table
 
-A grid table allows to have multiple lines per cells and allows to span cells over multiple columns. The following shows a simple grid table  
+A grid table allows having multiple lines per cells and allows spanning cells over multiple columns.
+The following shows a simple grid table:
 
-```
+```````````````````````````````` example
```

Comment:
I suspect this change may not be desirable; the other example blocks below appear to have both the source code and the rendered HTML. I suppose they're used for tests?

Reply:
Yeah, clearly this should not be an `example` block. I've reverted this change.